highfidelity/hifi
# Tool window in QML

Also includes some more polish to the window frames.

To test:  

* stop all scripts on your production build and exit.  
* launch the PR build
* verify you have no scripts running
* load the editing script from http://rawgit.com/jherico/hifi/tool_window/examples/edit.js
* click the button to go into edit mode.  

The tool window should pop up with the standard content.  It may take a moment to load for the first time.  The entity list, grid tool and entity property panes should be reasonably functional.

Because the tool window is now an overlay, *all* scripts that uses it need to be ported.  Aside from the edit related scripts, the only one in our codebase that uses the tool window is planky.js, which is also included in this PR
